types/part_set.go

```diff
@@ -264,3 +264,20 @@ func (ps *PartSet) StringShort() string {
 	defer ps.mtx.Unlock()
 	return fmt.Sprintf("(%v of %v)", ps.Count(), ps.Total())
 }
+
+func (ps *PartSet) MarshalJSON() ([]byte, error) {
+	if ps == nil {
+		return []byte("nil-PartSet"), nil
+	}
+
+	ps.mtx.Lock()
+	defer ps.mtx.Unlock()
+
+	return cdc.MarshalJSON(struct {
+		CountTotal    string        `json:"count/total"`
```

Comment:
why not make them 2 separate fields?

Reply:
Figured there's no need to take up the extra space but maybe that's silly (dump consensus is a big dump already). Also we're typically more interested in the ratio than we are in the two numbers